### PR TITLE
Fix quick start recommended requirements

### PR DESCRIPTION
**Pull request by dichotommy**
Thanks to @curquiza and @bidoubiwa for the feedback!

### CODE REVIEW

**Review by guimachiavelli — APPROVED** [review]
👍

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
 # Quick start
 
-This quick tour will help you get started with MeiliSearch in only a few steps. In order to make the most of it, you should be somewhat familiar with:
-
-- [The command line](https://www.learnenough.com/command-line-tutorial#sec-running_a_terminal)
-- [cURL](https://curl.se)
-- [JavaScript syntax](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics)
+This quick tour will help you get started with MeiliSearch in just a few steps. All that is required is a [command line](https://www.learnenough.com/command-line-tutorial#sec-running_a_terminal) with [cURL](https://curl.se) installed.
```
Comment (bidoubiwa) [inline learn/getting_started/quick_start.md:3]: You don't need curl if you are using an alternative way of downloading meilisearch. For example `brew install meilisearch` 

see: https://docs.meilisearch.com/learn/getting_started/installation.html

**Review by curquiza — APPROVED** [review]
Looks nicer! Thanks for taking my feedback into account ❤️

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**